styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java

```diff
@@ -170,6 +177,14 @@ void tick() {
     eligibleInstances.forEach(limitAndDequeue);
   }
 
+  private void updateMetrics(Map<String, Resource> resources,
+                             Map<String, Long> currentResourceUsage) {
+    resources.values().forEach(r -> stats.resourceConfigured(r.id(), r.concurrency()));
+    currentResourceUsage.forEach(stats::resourceUsed);
+    Sets.difference(resources.keySet(), currentResourceUsage.keySet())
+        .forEach(r -> stats.resourceUsed(r, 0));
```

Comment:
:star: 

Just it seems in the previous commits we treated globalConcurrency as well. Not anymore?